content/design-guidelines/color-usage.mdx

```diff
@@ -2,7 +2,7 @@
 title: Color usage
```

Comment:
The same URL needs fixing in more places:

https://github.com/search?q=repo%3Aprimer%2Fpresentations+color-system&type=code